features/bundler-cache/devcontainer-feature.json

```
@@ -0,0 +1,17 @@
+{
+  "id": "bundler-cache",
+  "version": "1.0.0",
+  "name": "Bundler cache",
+  "description": "Creates a volume for persisting the installed gems across different containers",
+  "containerEnv": {
+    "BUNDLE_PATH": "/bundle/vendor"
```

I noticed that once you're in the container, if you run `rails c` (or any other executable in your Gemfile) you'll get a "command not found" error. 

to solve this, we can add these env vars:

```suggestion
    "BUNDLE_PATH": "/bundle/vendor",
    "BUNDLE_BIN": "/bundle/bin",
    "PATH": "/bundle/bin:$PATH"
```

`BUNDLE_BIN` tells bundler to install the bins in the specified dir, and then we'll prepend this dir in the `PATH`.

By default, the `BUNDLE_BIN` would be something like `/bundle/vendor/ruby/COMPATIBILITY_VERSION/bin`. And compatibility version is not always the same ruby version you're running (for example i'm testing with 3.3.4 and the compat version is 3.3.0 - This is not trivial to get, so changing the `BUNDLE_BIN` makes sense..

---

I don't think we should add the bundle bin to the path like that. Instead people should do `bundle exec rails c` or `bin/rails c`. I expect global gems to be in the path but not bundled gems.